roles/bwc_repos/tasks/bwc_repos_apt.yml

```diff
@@ -0,0 +1,40 @@
+---
+
+- name: Assert that master_token is specified
+  fail: msg="License key must be supplied for BWC enterprise installation."
+  when: license is not defined
+
+- name: Install prereqs (Debian)
+  become: yes
+  apt:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - debian-archive-keyring
+    - apt-transport-https
+
+- name: Get GPG key URL for private repo
+  become: yes
+  shell: 'curl "{{ deb_gpg_key_url }}"'
+  register: deb_gpg_key
+
+# This is the exact key as the open source repo but this behavior might change. So just
+# following what's in packagecloud docs for private repos.
```

Comment:
Right, and it's not idempotent.

Please refactor `bwcrepos` role following the `st2repos` logic where possible.

Ex for this case: https://github.com/StackStorm/ansible-st2/blob/91bd7189500c36377e50b3cfaf20c4dbe6ef1f58/roles/st2repos/tasks/st2repos_apt.yml#L11-L16